doc/index.rst

```diff
@@ -49,6 +43,12 @@ Then, continue with the:
    tutorials
    Publications, Projects, and Tools <usage>
 
+.. figure:: _static/ix_features.svg
+   :width: 360px
+   :align: center
+
+   Features of ``ixmp``, ``message_ix``, and related packages(:cite:`huppmann_messageix_2018`)
```

Comment:
Space before left parenthesis, and period afterwards.

Reply:
Actually, if you look at the built docs, the citation style is numeric, so this comes out as “packages([3])”. I think the parentheses can be removed altogether.